server.js

```diff
@@ -1,8 +1,23 @@
 const express = require('express')
 const app = express()
+const mongoose = require('mongoose')
+const URI = 'mongodb+srv://dhruvk:q6GkQB4Kg3jvSHFd@dhruv-bike-info.dga5pzc.mongodb.net/?retryWrites=true&w=majority'
```

Comment:
Consider moving the database connection string to an environment variable to avoid exposing sensitive information. [important]